projects/ccf-shared/src/lib/components/body-ui/body-ui.component.ts

```diff
@@ -88,7 +108,9 @@ export class BodyUiComponent implements AfterViewInit, OnDestroy {
 
   private _interactive = true;
   private _rotation = 0;
+  private _rotationX = 0;
   private _zoom = 9.5;
+  private _target: [number, number, number] = [0, 0, 0];
```

Comment:
Rotation and target should be set at initialization time (there will be scenarios when they set target/rotation/etc initially that won't be picked up otherwise). See https://github.com/hubmapconsortium/ccf-ui/blob/6e26a38af610ea02ad520fba0578bb10a08d4bdc/projects/ccf-shared/src/lib/components/body-ui/body-ui.component.ts#L159